test/unit/components/Identity/processIdSyncs.spec.js

```diff
@@ -0,0 +1,56 @@
+import processIdSyncs from "../../../../src/components/Identity/processIdSyncs";
+import { cookie } from "../../../../src/utils";
+
+describe("Identity::processIdSyncs", () => {
+  const config = {};
+  const logger = {
+    log() {},
+    error() {}
+  };
+
+  const getControlObject = () => {
+    return JSON.parse(cookie.get("adobeIdSyncControl") || "{}");
+  };
+
+  it("tracks id syncs", done => {
+    const idSyncs = [
+      {
+        type: "url",
+        id: 411,
+        spec: {
+          url:
+            "//idsync.rlcdn.com/365868.gif?partner_uid=79653899615727305204290942296930013268",
+          hideReferrer: 0
+        }
+      }
+    ];
+
+    cookie.set(
+      "adobeIdSyncControl",
+      JSON.stringify({
+        123: Math.round(new Date().getTime() / 1000 / 60 / 60) - 10
+      }),
+      {
+        expires: 6 * 30 // 6 months
+      }
+    );
+
+    let obj = getControlObject();
+
+    expect(obj[123]).toBeDefined();
```

Comment:
I think you're testing your test? Probably unnecessary, but 🤷‍♂.

Reply:
I am doing that to show the difference in the same test later on after ID syncs are processed.